proxy/api/Cargo.toml

```diff
@@ -46,15 +46,15 @@ warp = { version = "0.3", default-features = false }
 
 [dependencies.radicle-daemon]
 git = "https://github.com/radicle-dev/radicle-link.git"
-rev = "87b3a077f9c821a1c21c8b79b98092239a373bde"
+rev = "bfdc3ec0c7ba542f2c9fc4f814f76fbd6fe6b2bc"
```

Comment:
One more commit got merged in link meanwhile.

Reply:
Updated it again.